src/components/Slider/Pit.js

```diff
@@ -30,8 +31,12 @@ const Pit = ({ key, style, children }) => {
 };
 
 Pit.propTypes = {
-  key: PropTypes.string,
```

Comment:
Why isn't the key needed anymore?

Reply:
Actually we used the `key` to bypass a bug with `preact`. Now it seems to render correctly, I  don't spot any bugs. I know @iam4x worked on it, if you think that something goes wrong don't hesitate :)

Reply:
It should be good, wasn't too sure about this fix I've made. It removed a warning and it was also used when I tried to moved the tests to a full preact rendering.